src/cryptoadvance/specter/server_endpoints/wallets/wallets_api.py

```diff
@@ -721,36 +721,16 @@ def is_address_mine(wallet_alias, address):
     )
 
 
-@wallets_endpoint_api.route("/wallet/<wallet_alias>/send/estimatefee", methods=["POST"])
+@wallets_endpoint_api.route("/wallet/<wallet_alias>/estimate_fee", methods=["POST"])
 @login_required
 def estimate_fee(wallet_alias):
-    """Returns a json-representation of a psbt which did not get persisted. Kind of a draft-run."""
+    """Returns the fee value from a created PSBT in dictionary form. The PSBT doesn't get persisted"""
     wallet: Wallet = app.specter.wallet_manager.get_by_alias(wallet_alias)
-    # update balances in the wallet
-    wallet.update_balance()
-    # update utxo list for coin selection
-    wallet.check_utxo()
-    if request.form.get("estimate_fee") != "true":
-        # Very critical as this form-value will prevent persisting the PSBT
-        return jsonify(
-            success=False,
-            error="Your Form did not specify estimate_fee = false. This call is not allowed",
-        )
-    psbt_creator = PsbtCreator(
-        app.specter,
-        wallet,
-        request.form.get("ui_option", "ui"),
-        request_form=request.form,
-        recipients_txt=request.form["recipients"],
-        recipients_amount_unit=request.form.get("amount_unit_text"),
```

Comment:
Seems to be a bit radical to purge that feature!

Reply:
Done in https://github.com/cryptoadvance/specter-desktop/pull/2164/commits/634259972079a2817e24729b3b63f2901747e86f